src/lib/timers/ingress_drop_monitor.lua

```diff
@@ -26,7 +27,10 @@ function new(args)
       tips_url = args.tips_url or default_tips_url,
       last_flush = 0,
       last_value = ffi.new('uint64_t[1]'),
-      current_value = ffi.new('uint64_t[1]')
+      current_value = ffi.new('uint64_t[1]'),
+      shm = {
+         ingress_packet_drop = {counter},
```

Comment:
I think this is probably not the right thing; this pattern is designed to be part of an app metatable's `shm` property, and Snabb takes care to `shm.open_frame` this frame.  Whereas here we don't have an app and indeed we have our own counter checking facility.  So maybe remove this `shm = {...}` block?